r/test/job_param.json

```diff
@@ -6,8 +6,8 @@
   "start_timestamp": "2020-01-01",
   "end_timestamp": "2020-12-01",
   "response": "class",
-  "samples": "samplePolygons.geojson",
-  "aoi": "aoi.geojson",
+  "samples": "samplePolygons.gpkg",
+  "aoi": "aoi.gpkg",
```

Comment:
Kann man die AOI aber auch weiterhin als .geosjon definieren?

Reply:
Ja kann man. Es wird direkt von read_sf gelesen. Dieser Funktion ist es egal ob .geojson oder .gpkg steht :) 
Da beiden in einen Spatial Dataframe umgewandelt wird ist das im späteren Verlauf des Programms identisch behandelt.